packages/strapi-admin/validation/user.js

```diff
@@ -12,6 +12,7 @@ const userCreationSchema = yup
     firstname: validators.firstname.required(),
     lastname: validators.lastname.required(),
     roles: validators.roles.min(1),
+    preferedLanguage: yup.string().notNull(),
```

Comment:
I think we might need to allow null if we want to allow users to not st the valu whn updating there profile. to b verified we a dev front